src/zenml/cli/__init__.py

```diff
@@ -693,6 +693,13 @@
 
 Note that you can only remove optional attributes.
 
+If you want to register secrets for all secret references in a stack, use the
```

Comment:
What does register mean here?

Reply:
Essentially uses the language of our secrets commands, `zenml secret register ...` registers a secret.